src/js/stores/__tests__/BaseStore-test.js

```diff
@@ -1,63 +1,37 @@
-jest.dontMock('../BaseStore');
-
 var BaseStore = require('../BaseStore');
 
 describe('BaseStore', function () {
 
   beforeEach(function () {
     this.instance = new BaseStore();
+    this.instance.on = jasmine.createSpy('eventOn');
```

Comment:
These are new tests. The old one test are moved to GetSetBaseStore-test.js and are unchanged.